src/controllers/import.js

```diff
@@ -156,6 +190,25 @@ function ImportController(context) {
         urls, options, importScript, customHeaders,
       } = multipartFormData;
 
+      const scopes = profile.getScopes();
+
+      // We only check if the URLs belong to the allowed domains if the user has the write scope
+      // We do not need to check the domains for users with scope: all_domains
```

Comment:
Since there is no check for the "write" scope for all_domains, then what happens if the user only has the ALL_DOMAINS scope and not the WRITE scope?